runtime/ms_rest_azure/lib/ms_rest_azure/azure_service_client.rb

```diff
@@ -19,6 +19,8 @@ def initialize(credentials, options = nil)
       @request_headers =  {
           'Content-Type' => 'application/json;charset=utf-8' # This is the current default for Azure services, and content-type supported by Autorest
       }
+      add_user_agent_information("ms_rest_azure/#{MsRestAzure::VERSION}")
+      add_user_agent_information("Azure-SDK-For-Ruby")
```

Comment:
are we including "Azure-SDK-For-Ruby" string for compatibility with already collected data? since it's getting logged together with ms_rest_azure info, does it give us additional benefits?